app/common/lib/randomUtil.js

```diff
@@ -0,0 +1,5 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports.random = () => Math.random()
```

Comment:
This was added specifically so we can stub the method with sinon (and verify it gets called)